commit c0530a22d7ad9934facfa46e5025dbfdee3237dd (refs/changes/50/1550/4)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-02-22T13:51:26-08:00 (2 years, 7 months ago)
    
    MANTA-3171 initial minimal doc rebranding updates for Triton Object Storage
    Reviewed by: David Pacheco <dap@joyent.com>

diff --git a/docs/bluejoy/header.html.in b/docs/bluejoy/header.html.in
index a24710f..0862b2a 100644
--- a/docs/bluejoy/header.html.in
+++ b/docs/bluejoy/header.html.in
@@ -6,7 +6,7 @@
 -->
 
 <!--
-    Copyright (c) 2014, Joyent, Inc.
+    Copyright (c) 2017, Joyent, Inc.
 -->
 
 <html lang="en">
@@ -40,7 +40,7 @@
 </head>
 <body>
 <div id="header">
-    <div class="logo"><a href="http://joyent.com"><img src="%(mediaroot)s/img/logo.svg"></a></div>
-    <h1>Manta Documentation</h1>
-    <a class="dev-center" href="http://joyent.com/developers"><span>For More Documentation and Support</span><br>Visit the Joyent Dev Center</a>
+    <div class="logo"><a href="https://joyent.com"><img src="%(mediaroot)s/img/logo.svg"></a></div>
+    <h1>Object Storage and Converged Analytics</h1>
+    <a class="dev-center" href="https://www.joyent.com/getting-started"><span>For More Documentation and Support</span><br>Visit the Getting Started page</a>
 </div>
